instrumentation/grpc/src/test/java/brave/grpc/BaseITTracingClientInterceptor.java

```diff
@@ -509,6 +489,59 @@ void initMessageTaggingClient() {
     assertThat(reporter.takeRemoteSpan(Span.Kind.CLIENT).parentId()).isNull();
   }
 
```

Comment:
NOTE: all ITs still pass, across dubbo and grpc. This means the data policy is exactly the same unless, someone initializes with RpcTracing, which will set the new policy from the rpc abstraction (ex "rpc.error_code")